lookout/style/typos/preprocessing.py

```diff
@@ -0,0 +1,30 @@
+from typing import Set
+
+import pandas
+
+from lookout.style.typos.utils import Columns, read_vocabulary
+
+
+def print_frequencies(tokens: Set[str], id_stats: pandas.DataFrame, frequency_column: str,
+                      path: str) -> None:
+    """
+    Print frequencies of tokens to a file.
+
+    Frequencies info is obtained from id_stats dataframe.
+    :param tokens: Set of tokens, for which frequencies should be printed.
+    :param id_stats: Dataframe with frequency information for tokens. It must be indexed by \
+                     tokens and containing columns `frequency_column`.
+    :param frequency_column: Name of the column in `id_stats`, from which to take frequency info.
+    :param path: Path to a .csv file to print frequencies to.
+    """
+    if isinstance(tokens, str):
+        tokens = set(read_vocabulary(tokens))
+    if isinstance(id_stats, str):
+        id_stats = pandas.read_csv(id_stats, index_col=Columns.Token)
+    frequencies = {}
+    for token, row in id_stats.iterrows():
+        if token in tokens:
+            frequencies[token] = row[frequency_column]
```

Comment:
```python
frequencies = {token: row[frequency_column] for token, row in id_stats.iterrows() 
               if token in tokens}
```

and I think something vectorized is also possible. if dataframe is indexed by tokens something like
```python
frequencies = id_stats.loc[tokens].dropna()[frequency_column]
...
```
should work. Also, I did not test it. `.dropna()` is not required if you know that all tokens from `tokens` can be found in dataframe.